io.github.Arian94.Tiny-beast-dictionary.yml

```diff
@@ -0,0 +1,31 @@
+app-id: io.github.Arian94.Tiny-beast-dictionary
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: tiny-beast-dict
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=home/.cache/${FLATPAK_ID}    #possibly needed if flatpak does not create local cache in app dir.
```

Comment:
are you sure it tries to use `~/.cache` ?

in that case to should be `--persist` and not `filesystem.

Reply:
Cache dir is a standard dir to cache stuff.
I don't know whether Flatpak creates its own ```.cache``` for each app inside ```${FLATPAK_DEST}/.cache``` or not.

Reply:
`XDG_CACHE_HOME` is set, so if the proper API is used there is nothing you need to do.

otherwise as I said, is `persist` instead.